src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const Newdays = days;
```

Comment:
no need to copy this variable
variables never start with a capital letter

Reply:
ok, thx